keyboards/bandominedoni/info.json

```diff
@@ -3,13 +3,35 @@
     "manufacturer": "3araht",
     "url": "https://github.com/3araht",
     "maintainer": "3araht",
+    "features": {
+        "midi": true,
+        "extrakey": true,
+        "encoder": true,
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "mousekey": false,
+        "nkro": false,
+        "rgblight": false,
+        "audio": false
+    },
+    "encoder": {
+        "rotary": []
+    },
```

Comment:
Encoders are enabled in the firmware, but have no configuration available (eg. pins defined)? Please fix that or remove encoder functionality from the firmware.

Reply:
The reason behind this is explained above. Somehow this works.
This split keyboard doesn't have rotary encoders on the left-hand side. But this empty statement is needed to avoid errors.

Reply:
I comfirm that https://github.com/qmk/qmk_firmware/pull/20179 will fix the problem.
I'll delete this.